packages/cli/src/internal/prompt/text.ts

```diff
@@ -58,19 +63,32 @@ function renderClearScreen(state: State, options: Options) {
 }
 
 function renderInput(nextState: State, options: Options, submitted: boolean) {
+  const text = getValue(nextState, options)
+
   const annotation = Option.match(nextState.error, {
-    onNone: () => submitted ? Ansi.white : Ansi.combine(Ansi.underlined, Ansi.cyanBright),
+    onNone: () =>
+      Match.value(submitted)
+        .pipe(
+          Match.when(true, () => Ansi.green),
```

Comment:
I don't think we should change the rendering of a submitted value. This should stay white as it was before.

Reply:
Whoops, did not mean to change the color. Good catch.